setup.py

```diff
@@ -21,5 +28,10 @@
         "osmnx",
         "geopandas",
         "s3fs",
+        "dask==2023.11.0",
```

Comment:
it might not be good to pin a specific version, it will fail if the user has another version. You can set a minimum though, like `dask>=2023.11.0`

Reply:
also, I think the conflict is from `odc-stac` above, not dask specifically. You can safely remove that, but will need to delete files that reference it

Reply:
`odc-stac` is used in [landsat_collection_2.py](https://github.com/wri/cities-cif/blob/2beecec8c561abd8baeec6958bd1a0900dc63d22/city_metrix/layers/landsat_collection_2.py#L2) and [sentinel_2_level_2.py](https://github.com/wri/cities-cif/blob/2beecec8c561abd8baeec6958bd1a0900dc63d22/city_metrix/layers/sentinel_2_level_2.py#L2). We probably cannot remove that without updating the layers.